cloudbuild/nightly.sh

```diff
@@ -24,36 +26,19 @@ fi
 readonly DATE="$(date +%Y%m%d)"
 readonly REVISION="0.0.${DATE}"
 readonly MVN="./mvnw -B -e -s /workspace/cloudbuild/gcp-settings.xml -Dmaven.repo.local=/workspace/.repository -Drevision=${REVISION}"
-readonly BUCKET="spark-lib-daily-snapshots"
 
 cd /workspace
 
 # Build
-$MVN install -DskipTests -Pdsv1_2.12,dsv2
+$MVN install -DskipTests -Pdsv1_2.11,dsv1_2.12,dsv2
```

Comment:
profile dsv1 contains both dsv1_2.11 and dsv1_2.12. Any reason for the change?

Reply:
so I made this change because on line 44 of nightly.sh here https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/master/cloudbuild/nightly.sh#L44 I see that we want to upload the 2.11 jar as well whereas in the command `$MVN install -DskipTests -Pdsv1_2.12,dsv2` we are only running the 2.12 connector and not 2.11. 

Should we change this to `$MVN install -DskipTests -Pdsv1,dsv2`?

Reply:
Changed it to dsv1, dsv2